shell/browser/api/electron_api_web_request.h

```diff
@@ -102,7 +97,7 @@ class WebRequest final : public gin_helper::DeprecatedWrappable<WebRequest> {
   void OnRequestWillBeDestroyed(extensions::WebRequestInfo* info);
 
  private:
-  WebRequest(v8::Isolate* isolate, content::BrowserContext* browser_context);
+  explicit WebRequest(base::PassKey<Session>);
```

Comment:
using `base::PassKey` seems unnecessary when the constructor is private, wouldn't it better to just friend this class in `api::session` ?

Reply:
https://github.com/electron/electron/pull/48762 makes the constructor public so this change was in preparation for it.